server/src/project/project.service.ts

```diff
@@ -620,7 +656,7 @@ export class ProjectService {
 
       dispositions: {
         ref: "dcp_communityboarddispositionid",
-        attributes: DISPOSITION_KEYS
```

Comment:
why not follow same pattern as PACKAGE and ARTIFACTs, and place this key inside their constant lists?

Reply:
Yeah, I think those other lists are probably fine but the problem is that they reference non-CRM attributes (like "documents") which we later learned in applicant portal was a problem for re-using them when building the CRM queries ($select=documents would give an error).